src/services/vehicle.ts

```diff
@@ -25,6 +25,16 @@ export const fetchFrameNumberTypes = async ({
       },
     })
 
+    if (!typeIds.length) {
+      return {
+        tag: "success",
+        result: {
+          content: [],
+          pagination: null,
+        },
+      }
+    }
+
     const types = await fetchTypes({
```

Comment:
Is `fetchTypes` really supposed to return all types if we query for an empty list of ids?

Reply:
Yes, this is how BE implemented it, check this slack discussion for more details https://autoricardo-ag.slack.com/archives/C8RRP2UQ4/p1624284136079200